src/app/public/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -0,0 +1,57 @@
+<label
+  class="sky-date-range-picker-label"
+>
+  {{ label }}
+  <select
```

Comment:
Should these select fields have some sort of `aria` labels?

Reply:
aria label is only necessary if there isn't going to be a label associated with the input. This setup should associate this parent label with the select field and the label input is going to be required in our documentation (though not enforced with an error).

Reply:
👍 great answer.